src/app/shared/shared.module.ts

```diff
@@ -31,12 +31,12 @@ const dotEventSocketURLFactory = () => {
 
 @NgModule({
     declarations: [],
-    imports: [CommonModule, DotDropdownModule, GravatarModule, MainNavigationModule],
+    imports: [CommonModule, DotDropdownModule, DotGravatarModule, MainNavigationModule],
     exports: [
         CommonModule,
         // Common Modules
         DotDropdownModule,
-        GravatarModule,
+        DotGravatarModule,
```

Comment:
If we turn this into a module, we can just call this module inside the toolbar and we don't have to put it here in the share module.